testslide/lib.py

```diff
@@ -74,12 +68,18 @@ def _validate_function_signature(
             else:
                 argname = argspec.args[idx]
             try:
-                _validate_argument_type(argspec.annotations, argname, args[idx])
+                expected_type = argspec.annotations.get(argname)
+                if not expected_type:
+                    continue
```

Comment:
Why move this previous lines out of the `_validate_argument_type` function? it looks like we have them everywhere we use that function no?

Reply:
Because from StrictMock, we don't have argspec, only the type available. By doing this change here, we can use `_validate_argument_type` for function arguments and and class attributes type validation :-D